.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - sed -i 's/KEY_VALUE_STORE_PASSWORD=/KEY_VALUE_STORE_PASSWORD=redis/' .env
   - sed -i 's/MAP_SERVER_ADMIN_PASSWORD=/MAP_SERVER_ADMIN_PASSWORD=geoserver/' .env
   - cp django/publicmapping/config/config.dist.xml django/publicmapping/config/config.xml
-  - ./scripts/update
+  - ./scripts/setup
```

Comment:
For your errors above, I wonder if you need to do the couple steps mentioned in the README:
```
$ cp django/publicmapping/config/config.dist.xml django/publicmapping/config/config.xml
$ # Copy .env file and add password/modify as necessary
$ # NOTE: Leave MAP_SERVER_ADMIN_PASSWORD the same for now
$ cp django/publicmapping/.env.sample django/publicmapping/.env
```

Reply:
I wonder if it might be helpful to have a `Makefile` that we could use where the targets would have dependencies on these configuration files that need to be in place to run the app.

Alternatively, we could make the `settings.py` file a bit smarter by catching exceptions and outputting more helpful messaging when `docker-compose up` fails (eg. "Did you forget to do a build?").

Reply:
@tnation14 good call; I had forgotten we had our own STRTA ADR and was going off the GitHub spec, which includes a `bootstrap` script, whereas ours doesn't. I've shifted everything back to use `setup` and `update`.

Reply:
@pcaisse I've gone ahead and added a try/except to settings which prints a more helpful message if `config_settings.py` isn't found.

Reply:
Turns out that `config_settings.py` was getting overridden by `docker-compose` volume mounts, so Django couldn't find the file during initialization. I was able to get the script to succeed from a clean environment by adding this command to `scripts/update`:

```diff
diff --git a/scripts/update b/scripts/update
index 6301c3bc..bda76076 100755
--- a/scripts/update
+++ b/scripts/update
@@ -31,6 +31,14 @@ then
         usage
     else
         build_containers
+
+        # Write config_settings to the filesystem so it doesn't get
+        # overwritten by docker-compose volume mounts
+        docker-compose run --rm --entrypoint "bash -c" django \
+            "python -m district_builder_config.generate_settings \
+                config/config.xsd \
+                config/config.xml \
+                publicmapping/config_settings.py"
         docker-compose up -d postgres django
         run_migrations
     fi
```

Reply:
Cool, it seems this is the same issue discussed here: https://github.com/PublicMapping/DistrictBuilder/pull/512#pullrequestreview-96657452

That's great that you likely came up with a fix.

Reply:
I'm going to add this into this PR since it seems to be afflicting Travis.